docs/book/src/user/quick-start.md

```diff
@@ -38,6 +38,8 @@ export KUBECONFIG=<...>
 
 **Minimum [kind] supported version**: v0.7.0
 
+**If you are using v0.8.0+ you must export the variable `KIND_EXPERIMENTAL_DOCKER_NETWORK`**
```

Comment:
What about merging the two changes in the quick start page?

**If you are using v0.8.0+ and want to use the docker provider**: 
```bash
  export KIND_EXPERIMENTAL_DOCKER_NETWORK=bridge
```

Reply:
added tab to split 0.7.x and 0.8.x setup